dev-util/cutter/cutter-1.10.1.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit qmake-utils xdg-utils python-single-r1
+
+DESCRIPTION="A Qt and C++ GUI for radare2 reverse engineering framework"
+HOMEPAGE="https://www.radare.org"
+SRC_URI="https://github.com/radareorg/cutter/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="jupyter webengine"
+REQUIRED_USE="webengine? ( jupyter )
+	${PYTHON_REQUIRED_USE}"
+
+DEPEND="
+	${PYTHON_DEPS}
+	>=dev-qt/qtcore-5.9.1:5
+	>=dev-qt/qtgui-5.9.1:5
+	>=dev-qt/qtsvg-5.9.1:5
+	>=dev-qt/qtwidgets-5.9.1:5
+	dev-qt/qtnetwork:5
+	>=dev-util/radare2-4.2.1
+	jupyter? ( dev-python/jupyter )
```

Comment:
Does this perhaps require the same python interpreter to be used as in this ebuild? Ie, if I have jupyter installed for python3.7, will it work with 3.6 on this one?

Reply:
Jupyter here makes no sense at all, it has been moved to a plugin long ago: https://github.com/radareorg/cutter-jupyter

Reply:
I removed the jupyter & webengine use flags